published/external/xdpapi_helper.h

```diff
@@ -0,0 +1,279 @@
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <xdpapi.h>
+
+#if defined(_KERNEL_MODE)
+
+#include <wdm.h>
```

Comment:
I highly recommend _not_ including wdm.h, because the major Windows driver headers (wdm.h, ntddk.h, ntifs.h) are catastrophically difficult to include correctly. It is conceptually the correct thing to do.

Reply:
I forget why I added wdm.h, but it was needed at that time. However now it can be removed without issue. interesting